src/models.py

```diff
@@ -9,6 +11,21 @@
 logger = logging.getLogger(__name__)
 
 
+def get_class_hash(cls) -> str:
```

Comment:
thought: I can't see us using this anywhere else so maybe this makes sense as a class method?

Reply:
i guess, but i thought it might be useful for other models another time. so might as well leave it separate!